webpack/assets/javascripts/react_app/components/PF4/InlineEdit/InlineEdit.stories.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { withKnobs, text, boolean } from '@theforeman/stories';
+import Story from '../../../../../../stories/components/Story';
+import InlineEdit from './InlineEdit';
+
+export default {
+  title: 'Components|PF4/InlineEdit',
```

Comment:
```suggestion
  title: 'Components/PF4/InlineEdit',
```
When #7990 is merged these will all need to be `/`; for now they all need to be `|`.